crm-client/src/pages/projects/ProjectsInsert.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react'
+import { useForm } from "react-hook-form";
+import api from '../../api'
+
+import styled from 'styled-components'
+
+import { nameValidator } from '../../services/validation-rules'
+
+const Title = styled.h1.attrs({
+    className: 'h1',
+})``
+
+const Wrapper = styled.div.attrs({
+    className: 'form-group',
+})`
+    margin: 30px 80px;
+`
+
+const Label = styled.label`
+    margin: 5px;
+`
+
+const InputText = styled.input.attrs({
+    className: 'form-control',
+})`
+    margin: 5px;
+    width: 30%;
+`
+
+const Button = styled.button.attrs({
+    className: `btn btn-primary`,
+})`
+    margin: 15px 15px 15px 5px;
+`
+
+const CancelButton = styled.a.attrs({
+    className: `btn btn-danger`,
+})`
+    margin: 15px 15px 15px 5px;
+`
+
+function ProjectsInsert() {
+    const { register, handleSubmit, formState: { errors } } = useForm();
+
+    const handleCreateProject = useCallback(async (data) => {
+        const { name } = data
+        const payload = { name }
+
+        try {
+            await api.insertProject(payload);
+
+            window.alert(`Project inserted successfully`)
```

Comment:
Nice to change all alerts to [react-toastify](https://www.npmjs.com/package/react-toastify)